src/wp-includes/functions.php

```diff
@@ -9090,3 +9090,139 @@ function wp_is_heic_image_mime_type( $mime_type ) {
 
 	return in_array( $mime_type, $heic_mime_types, true );
 }
+
+
+/**
+ * Outputs the tooltip for a specific field.
+ *
+ * @since 6.7.0
+ *
+ * @param string $field_id             The unique identifier for the tooltip container.
+ * @param string $tooltip_text         The text content to be displayed in the tooltip.
+ * @param string $tooltip_button_label Optional. The label for the tooltip button. Default is 'Help'.
```

Comment:
Change the default value to an empty string, and set the default as `__( 'Help' )` inside the function, so that the default value is translatable.